src/domain.jl

```diff
@@ -2027,31 +2025,31 @@ function calcElemShapeFunctionDerivatives( x, y, z,
     b[6,3] = -b[4,3]
     b[7,3] = -b[1,3]
     b[8,3] = -b[2,3]
+    end #inbounds
 
     # calculate jacobian determinant (volume)
     el_volume = 8.0 * ( fjxet * cjxet + fjyet * cjyet + fjzet * cjzet)
-    return nothing
+    return SMatrix(b), el_volume
```

Comment:
@michel2323 this is an important change. I think in the Fortran version `el_volume` was passed in as reference and thus updated outside the function